deploy/deploy.go

```
@@ -7,6 +7,8 @@ import (
 	"github.com/timmyjinks/tysoncloud-cli/store"
 	"github.com/timmyjinks/tysoncloud-cli/util"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

golangci-lint (2.12.2)

[error] ST1019(related information): other import of "k8s.io/api/core/v1"

(staticcheck)